src/main/java/uk/gov/hmcts/reform/pip/data/management/service/artefact/ArtefactDeleteService.java

```diff
@@ -122,4 +139,40 @@ public void deleteArtefactById(String artefactId, String issuerId) {
     private void triggerThirdPartyArtefactDeleted(Artefact deletedArtefact) {
         log.info(writeLog(subscriptionManagementService.sendDeletedArtefactForThirdParties(deletedArtefact)));
     }
+
+    @Transactional
```

Comment:
I think we can remove the transactional here

Reply:
removed.